### ISSUE TITLE
ARToolkit support

### ISSUE THREAD

**Issue by kaworuyoshi** [Rajawali/Rajawali#1731]
#### Suggestion
Add support for the ARToolkit sdk, since vuforia lincensing has gone crazy.

**Comment by jwoolston** [comment]
I believe this is the alternative @MasDennis was proposing.

**Comment by aalices** [comment]
Is it possible to integrate current version of RajawaliVuforia with the library at version 5? I need this for a non-commercial project and would be really greatful for any hint

**Comment by jwoolston** [comment]
It is, but someone needs to do the work to upgrade the API.

**Comment by aalices** [comment]
I actually made some progress to do this and the only but severe problem is that my camera preview is upside down. Do you have any idea what causes that?

**Comment by jwoolston** [comment]
Texture coordinates most likely. Are you using `ScreenQuad`?

**Comment by aalices** [comment]
Yes, I almost haven't changed the Java code (mainly changing from QCAR to Vuforia). I've recently noticed that the same issue happened when still using older Vuforia and the newest Rajawali distr. Downgrading to 1.0.0-SNAPSHOT solved the issue.

**Comment by jwoolston** [comment]
If you have code for integrating AR toolkit you wish to share, you can always PR it.

**Comment by aalices** [comment]
I've actually been talking about integrating to Vuforia 5 :)
I have to make the project less messy and will publish it. Now I have a problem with object slightly moving around when for example rotating camera. I've read that it can be related to camera's viewport/fov differences with some QCAR values. Still looking for the cause and solution.

**Comment by jwoolston** [comment]
Either would be good. Im sure the licensing terms of vuforia 5 would be acceptable to some users.
